airflow/www/app.py

```diff
@@ -74,17 +76,37 @@ def sync_appbuilder_roles(flask_app):
         flask_app.appbuilder.sm.sync_roles()
 
 
+def strip_tags(input_html: str) -> str:
```

Comment:
```suggestion
def _strip_tags(input_html: str) -> str:
```

since we don’t intent to import this anywhere else.